keyboards/kc60se/keymaps/default/keymap.c

```diff
@@ -13,16 +13,35 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+
 #include QMK_KEYBOARD_H
 
-#define _DEFAULT 0
-#define _FN 1
+/* Happy Hacker Keyboard */
+
+enum layer_names { _BL, _FL };
 
-#define xxxxxx KC_TRNS
+#define _______ KC_TRNS
```

Comment:
QMK defines this automatically.

https://github.com/qmk/qmk_firmware/blob/58101cbb73a16678280d8a0cea275d680b603e58/quantum/quantum_keycodes.h#L29-L31

```suggestion
```

Reply:
I change to 7_ but forgot to delete the define. thanks